pkg/utils/utils.go

```diff
@@ -133,7 +151,7 @@ func (u *utils) RunCommandOnPod(pod *corev1.Pod, command []string) ([]byte, erro
 		Stdin:  os.Stdin,
 		Stdout: &buf,
 		Stderr: os.Stderr,
-		Tty:    true,
+		Tty:    false,
```

Comment:
missing the change in VersionedParams we discussed

Reply:
i dont think all those needed 
like we are running the command on create now and this change is adding weird warning on the logs